apps/wallet/src/background/accounts/index.ts

```diff
@@ -71,6 +72,17 @@ export async function isAccountsInitialized() {
 	return (await (await getDB()).accounts.count()) > 0;
 }
 
+export async function getAccountsStatusData(
+	accountsFilter?: string[],
+): Promise<Required<WalletStatusChange>['accounts']> {
+	const allAccounts = await (await getDB()).accounts.toArray();
```

Comment:
nit: can make this a bit more concise

```typescript
export async function getAccountsStatusData(
	accountsFilter?: string[],
): Promise<Required<WalletStatusChange>['accounts']> {
	const allAccounts = await (await getDB()).accounts.toArray();
	return allAccounts
		.filter(({ address }) => !accountsFilter?.length || accountsFilter.includes(address))
		.map(({ address, publicKey }) => ({ address, publicKey }));
}
```

Reply:
By doing that there is a case to do some unnecessary iterations, that's why I did it like this. It doesn't really matter because of the length of the `allAccounts` (I would assume < 10 usually) - I will update it